examples/Cargo.toml

```diff
@@ -11,13 +11,31 @@ description = """
 AA (ERC-4337) bundler examples
 """
 
-[dev-dependencies]
-
+[dependencies]
+alloy-primitives = "0.3"
+alloy-sol-types = "0.3"
 anyhow = "1"
 ethers = { version = "2.0.4" }
+reqwest = { version = "0.11.4", features = ["json"] }
+serde = "1"
+serde_json = "1"
+silius-contracts = { path = "../crates/contracts" }
 silius-primitives = { path = "../crates/primitives" }
+silius-tests = { path = "../tests" }
 tokio = { workspace = true }
 
 [[example]]
 name = "user-operation"
 path = "user_operation.rs"
+
+[[example]]
+name = "create_with_factory"
```

Comment:
I would rename to simple_account_create, simple_account_transfer, simple_account_deposit ... if we add more examples in the future